nss-day-cohort-38/bangazon-workforce-management-aquarii-star-system
# Kp training edit

## Description


Training Programs Details Page:
- Added list of employees attending
- Added Edit button
- Added note on past events that they cannot be edited or deleted
- Added PUT handling for program edit form

Training Programs Form:
- Added edit functionality
- created required url route
- updated views package

## How do I access this branch?

1. `git checkout master`
1. `git fetch --all`
1. `git checkout kp-training-edit`
1. `git pull origin kp-training-edit` (in case you've fetched since last I pushed up)

## Motivation and Context


#11 

## How Has This Been Tested?



1. `http://127.0.0.1:8000/training_programs`

Upcoming Program:
1. Click on an "upcoming" program. 
1. Check that there are: 
-- Note: *events in the future cannot be edited or deleted*
-- A list of employees attending

Past Program:
1. Navigate back to the list. 
1. Click on a "past" program.
1. Check that there are:
-- A list of employees attending
-- edit and delete buttons
1. Click edit
1. The form should populate with the training_program details
1. Edit them and save. 
1. Click again on the program to confirm the changes were saved.

## Screenshots (if appropriate):
Pew pew!

## Types of changes


## Checklist:
